src/extension.ts

```diff
@@ -201,7 +201,44 @@ function configureExpandMacro(context: ExtensionContext) {
     );
     panel.webview.html = getExpandMacroWebviewContent(res);
   });
-  
+
+  context.subscriptions.push(disposable);
+}
+
+function configureManipulatePipes(context: ExtensionContext, operation: "toPipe" | "fromPipe") {
+  const commandName = `extension.${operation}`;
+
+  const disposable = vscode.commands.registerCommand(commandName, async () => {
+    const extension = vscode.extensions.getExtension("jakebecker.elixir-ls");
+    const editor = vscode.window.activeTextEditor;
+    if (!extension || !editor) {
```

Comment:
Is the extension check necessary?

Reply:
Not exactly, this just fails faster upon a bizarre error, if that makes sense. I can remove it if you prefer